README.md

```diff
@@ -1,2 +1,67 @@
-# awesome-books
-This project includes a basic website that allows users to add/remove books from a list by using JavaScript objects and arrays. Modified dynamically by the DOM and added basic events.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+> This is a new website feature for this new Module
```

Comment:
- [OPTIONAL] Please kindly give a meaningful project title, as this will make your readme file more informative and professional.